### PR TITLE
Fly camera control

### PR DESCRIPTION
**Pull request by SylvainCorlay**
The same as the one used in the lightning talk at scipy plus some cleanup.

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
@jasongrout

File: pythreejs/nbextension/pythreejs.js [hunk 1]
```diff
@@ -372,14 +374,69 @@ define(["widgets/js/widget", "widgets/js/manager", "base/js/utils", "underscore"
...
+                }, that);
+                that.obj = new THREE.FlyControls(that.controlled_view.obj, that.options.dom);
+                that.register_object_parameters();
+                that.options.register_update(that._update, that);
+                that.obj.addEventListener('change', that.options.render_frame);
+                that.obj.addEventListener('change', function() { that.update_controlled(); });
+                that.options.start_update_loop();
+                that.model.on_some_change(['forward_speed', 'upward_speed', 'lateral_speed',
+                                           'roll', 'yaw', 'pitch'], that.update_plane, that);
+                // if there is a three.js control change, call the animate function to animate at least one more time
```
Comment (jasongrout) [inline pythreejs/nbextension/pythreejs.js]: delete this comment.

**Comment by SylvainCorlay** [comment]
Done.

**Comment by jasongrout** [comment]
Thanks!

**Pull request merged**